### PR TITLE
add v0.20.2 blogpost

### PR DESCRIPTION
**Pull request by narimiran**

### CODE REVIEW

File: jekyll/_posts/2019-07-17-version-0202-released.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+The Nim team is happy to announce version 0.20.2, which is our second
+release candidate for version 1.0.
+
+To read more about version 0.20 and our plans for version 1.0 and beyond,
+check the [previous release notes](https://nim-lang.org/blog/2019/06/06/version-0200-released.html).
+This is mostly a bugfix release of version 0.20.0 (1.0 RC1).
+
+The only new feature is that `toOpenArray`, our slicing operator,
+is now available for JavaScript backend.
```
Comment (dom96) [inline jekyll/_posts/2019-07-17-version-0202-released.md]:
```suggestion
is now available for the JavaScript backend.
```

File: jekyll/_posts/2019-07-17-version-0202-released.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+Nim now bootstraps with `--styleCheck:error` flag, meaning that the compiler
+and the standard library are now style checked and they won't compile if
+different styles (e.g. `fooBar` and `foo_bar`) are used.
+If you would like to style check your own nimble package, but not its
+dependencies, you can use `--styleCheck:hint`, making your package consistent
```
Comment (dom96) [inline jekyll/_posts/2019-07-17-version-0202-released.md]: So I'll get hundreds of hints from the packages I depend on as well?

File: jekyll/_posts/2019-07-17-version-0202-released.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+release candidate for version 1.0.
+
+To read more about version 0.20 and our plans for version 1.0 and beyond,
+check the [previous release notes](https://nim-lang.org/blog/2019/06/06/version-0200-released.html).
+This is mostly a bugfix release of version 0.20.0 (1.0 RC1).
+
+The only new feature is that `toOpenArray`, our slicing operator,
+is now available for JavaScript backend.
+
+Nim now bootstraps with `--styleCheck:error` flag, meaning that the compiler
```
Comment (dom96) [inline jekyll/_posts/2019-07-17-version-0202-released.md]: Is this not also a new feature? Was ``-styleCheck:error`` in 0.20.0?

File: jekyll/_posts/2019-07-17-version-0202-released.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+
+## Library additions
+
+- `toOpenArray` is now available for the JS target.
+
+
+## Library changes
+
+- Fix async IO operations stalling even after socket is closed. (#11232)
```
Comment (dom96) [inline jekyll/_posts/2019-07-17-version-0202-released.md]: 🚀

File: jekyll/_posts/2019-07-17-version-0202-released.md (hunk 3 above)
Comment (Araq) [reply jekyll/_posts/2019-07-17-version-0202-released.md]: Yeah, but it was so broken that it was effectively unusable.

File: jekyll/_posts/2019-07-17-version-0202-released.md (hunk 2 above)
Comment (Araq) [reply jekyll/_posts/2019-07-17-version-0202-released.md]: No, you only get hints for the "main" nimble package/program that you compile.

**Pull request merged**